src/content/developers/docs/scaling/state-channels/index.md

```diff
@@ -6,71 +6,256 @@ sidebar: true
+
+State channels don't have the same security guarantees. To some extent, a state channel is a miniature version of Mainnet...with two nodes. With a limited set of participants enforcing rules, the possibility of malicious behavior (e.g., proposing invalid state updates) increases. To ensure security, state channels rely on a dispute arbitration system based on [fraud proofs](/glossary/#fraud-proof). 
+
+## How do state channels interact with Ethereum? {#how-do-state-channels-interact-with-ethereum}
+
+Although they exist as off-chain protocols, state channels have an on-chain component: the smart contract deployed on Ethereum when opening the channel. This contract controls the assets deposited into the channel, verifies state updates, and arbitrates disputes between participants. 
+
+State channels don't publish transaction data or state commitments to Mainnet, unlike [layer 2](/layer-2/) scaling solutions. However, they are more connected to Mainnet than, say, [sidechains](/developers/docs/scaling/sidechains/), making them somewhat safer.  
+
+State channels rely on the main Ethereum protocol for the following:
+
+### 1. Liveness {#liveness}
+The on-chain contract deployed when opening the channel is responsible for the channel's functionality. If the contract is running on Ethereum, then the channel is always available for usage. Conversely, a sidechain can always fail, even if Mainnet is operational, putting user funds at risk. 
+
+### 2. Security {#security}
+
+To some extent, state channels rely on Ethereum to provide security and protect users from malicious peers. As discussed in later sections, channels use a fraud proof mechanism that lets users challenge attempts to finalize the channel with an invalid or stale update. 
+
+In this case, the honest party provides the latest valid state of the channel as a fraud proof to the on-chain contract for verification. Fraud proofs enable mutually distrustful parties to conduct off-chain transactions without risking their funds in the process. 
+
+### 3. Finality {#finality}
+
+State updates collectively signed by channel users are considered as good as on-chain transactions. Still, all in-channel activity only achieves true finality when the channel is closed on Ethereum. 
+
+In the optimistic case, both parties can cooperate and sign the final state update and submit on-chain to close the channel, after which the funds are distributed according to the channel's final state. In the pessimistic case, where someone tries to cheat by posting an incorrect state update on-chain, their transaction isn't finalized until the challenge window elapses.
+
+## How do state channels work? {#how-do-state-channels-work}
```

Comment:
It seems strange to me to have a section "How do state channels work" immediately after two chunky paragraphs that  already explain some aspects of how state channels work. Feels to me like this section needs a restructure so that "How state channels work" is a top level header, with several subheaders. "How do state channels interact with Ethereum" should be a subheader, for example. Please also try to minimize repetition of information.

Reply:
I applied the same structure used in the other scaling pages here: High-level overview of the protocol → description of its relationship with Ethereum → low-level explanation of the protocol's mechanics, etc. 

Maybe the chunkiness of preceding paragraphs is because I wanted to keep the original content (explanations of channels, payment channels, and state channels). I just edited the text to have the "How do state channels interact with Ethereum?" as an H4 header under the "How do state channels work?" section, though. If there are any other parts you want to see placed in the section, let me know.

Reply:
yea I understand and appreciate what you've done - the level of detail is great and overall the page is improved. No doubt. My concerns are just a) whether the additional content makes the existing structure feel imbalanced and maybe disrupts the logic flow through the page, and b) the level of assumed prior knowledge required to parse some of the more complex sentences that arrive early. The original structure may have worked well with the original content but may need some refining now.

I don't think these need to be big structural changes necessarily - perhaps just making sure technical terms are defined on first use (even just by linking out to an external definition) and tweaking the headers would be enough (e.g. instead of "how does it work" we might change to "finer details" or "diving deeper" or something), just to help scaffold the reader a little more.

Reply:
Sure, the comment about structure makes sense. On linking to definitions of technical terms: I think @minimalsm commented about too many links in a page affecting reader experience (this was for the data availability page). 

Still, it's a good idea to provide a gentler on-ramp for beginner-level readers by including links to definitions. That's partly why I'm working on including some new definitions for the glossary--stuff like "Merkle proof" or "state transition" may sound like Greek to new readers, but having them in the glossary (and linking to it) can help.